tools/biobb_pytorch/biobb_apply_mdae.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="biobb_pytorch_apply_mdae" name="ApplyMdae" version="4.2.1" >
```

Comment:
You can define a token here which means you can avoid repeating the version number, e.g. https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/blob/3c6707a93f43fa435d290faaec728d730d8289a5/tools/biomoldyn/NEQGamma.xml#L4

We recommend this because it allows us to keep tool wrappers up to date automatically with the latest software releases as well.

Reply:
THX fixed and committed